src/d3a/d3a_core/sim_results/__init__.py

```diff
@@ -29,6 +33,23 @@ def is_buffer_node_type(area):
     return area['type'] == "InfiniteBusStrategy"
 
 
+def get_unified_area_type(area):
+    if is_pv_node_type(area):
+        return "PV"
+    elif is_load_node_type(area):
+        return "Load"
+    elif is_cell_tower_type(area):
+        return "CellTower"
+    elif is_prosumer_node_type(area):
+        return "Storage"
+    elif is_bulk_power_producer(area) or is_buffer_node_type(area):
+        return "MarketMaker"
+    elif area['type'] == "FinitePowerPlant":
```

Comment:
Even if it is just one value, I would prefer it if you created one more function that checks the finite power plant, in order to be more consistent.

Reply:
Okay, will do in second round, as it is cosmetics and doing it as part of this PR will further delay frontend.